sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -1254,8 +1254,9 @@ inline pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
     PI_TO_UR_MAP_DEVICE_INFO(
         PI_EXT_ONEAPI_DEVICE_INFO_INTEROP_SEMAPHORE_EXPORT_SUPPORT,
         UR_DEVICE_INFO_INTEROP_SEMAPHORE_EXPORT_SUPPORT_EXP)
-    PI_TO_UR_MAP_DEVICE_INFO(PI_EXT_INTEL_DEVICE_INFO_ESIMD_SUPPORT,
-                             UR_DEVICE_INFO_ESIMD_SUPPORT)
+    // TODO: Re-enable when UR 'adapters' branch is updated

```

Comment:
why do we need this change?

Reply:
This is just a temporary fix until the `adapters` branch in the UR repo is up to date, currently it's quite a few commits behind `main` so is missing this enum value (see https://github.com/oneapi-src/unified-runtime/pull/881). I'm planning on keeping this in draft until that is resolved and the L0 adapter source is actually merged.

Reply:
This has been reverted now